atlp-rwanda/script-heroes-frontend
# #174797474 Avail Requests for Approval

**What does this PR do?**
This PR displays the manager's requests that are associated with him

**Description of Task to be completed?**
- create a component for displaying all requests
- create pagination to display a few requests per page
- test the feature 

**How should this be manually tested?**

- clone this repo `git clone https://github.com/atlp-rwanda/script-heroes-frontend`
- checkout the branch git checkout `avail-requests-#174797474 `
- install dependencies` yarn install`
- start the app yarn run dev
- got to` /request/manager` to display all requests 
- check responsiveness on different screen sizes

**Any background context you want to provide?**

> set the token in local storage `x-auth-token: value`

**What are the relevant pivotal tracker stories?**
   [#174797474](https://www.pivotaltracker.com/n/projects/2465784/stories/174797474)

**Screenshots (if appropriate)**
![noreq1](https://user-images.githubusercontent.com/50476344/102271402-0e887e00-3f28-11eb-916e-919a0b07b875.png)
![noreq](https://user-images.githubusercontent.com/50476344/102271410-0fb9ab00-3f28-11eb-8fce-0280e1bf5540.png)
